test/stablex/batch_exchange_viewer.js

```diff
@@ -20,6 +20,64 @@ contract("BatchExchangeViewer", accounts => {
     await batchExchange.addToken(token_2.address)
   })
 
+  describe("getBalances", () => {
+    it("returns all listed tokens", async () => {
+      const viewer = await BatchExchangeViewer.new(batchExchange.address)
+      const result = await viewer.getBalances(accounts[0])
+      assert.equal(result.filter(balance => balance.availableSellAmount == 0).length, 3)
```

Comment:
https://github.com/gnosis/dex-contracts/blob/ee926aca132b284051db53c40ab5e53ed439c255/test/stablex/batch_exchange_viewer.js#L13

The contract gets initiated with 3 tokens for each test. I assume the balances are all zero for account 0.

Reply:
Good point, since this detail is very relevant for the test, I should probably do the instantiation in the test case instead of relying on people reading the before each block (at least for this one)

Reply:
I would be cool with just a comment.